web-registry/src/components/organisms/CreditBatches.tsx

```diff
@@ -40,13 +40,13 @@ interface HeadCell {
 }
 
 const headCells: HeadCell[] = [
+  { id: 'start_date', numeric: true, label: 'date created' },
```

Comment:
This is in the comps, but we removed it because `start_date` != 'date created'. "Date created" is not yet available in the blockchain data. 
```suggestion
```

Reply:
thanks!